apps/fhir/server/authentication.py

```diff
@@ -63,9 +98,17 @@ def search_fhir_id_by_identifier(search_identifier, request=None):
         + "Patient/?identifier=" + search_identifier \
         + "&_format=" + settings.FHIR_PARAM_FORMAT
 
+    s = requests.Session()
+
+    req = requests.Request('GET', url, headers=headers)
+    prepped = req.prepare()
+    # add fhir data audit logging
+    pre_fetch.send_robust(FhirServerAuth, request=req)
     # Get FHIR service backend response.
-    #   TODO: Should work with verify=True
-    response = requests.get(url, cert=certs, headers=headers, verify=False)
+    # TODO: Should work with verify=True
+    # response = requests.get(url, cert=certs, headers=headers, verify=False)
```

Comment:
@JFU-GIT This comment maybe should be removed, if you agree. Might be good to keep for reference?

Reply:
there is BB2-245 track that, so it can be removed.